inst/pandoc_templates/jolla-blue.html

```diff
@@ -12,6 +12,21 @@
     <style type="text/css">body {background-color: #3A66B7;font-family: 'Merriweather', serif;color: #FEFEFA;}</style>
 
     <title>$title$</title>
+    
+    $if(meta)$
+    <meta name="description" content="$meta.description$"></meta>
+      <meta name="twitter:description" content="$meta.description$"></meta>
+      <meta name="og:description" content="$meta.description$"></meta>
+      <meta name="twitter:title" content="$title$"></meta>
+      <meta name="og:title" content="$title$"></meta>
+      <meta name="twitter:url" content="$meta.url$"></meta>
+      <meta property="og:url" content="$meta.url$"></meta>
+      <meta name="twitter:image:src" content="$meta.url$/$image$"></meta>
```

Comment:
I think it's worth allowing users to specify the share card image URL directly, maybe something like

```suggestion
      <meta name="twitter:image:src" content="$if(meta.image)$$meta.image$$else$$meta.url$/$image$$end$"></meta>
```

this would allow users to specify a _complete URL_ to their social image, if desired, or fall back to url + image otherwise.

```yaml
meta:
  description: "floatplane data artist"
  url: "example.com"
  twitter: "yourname"
  image: https://example.com/social.png

# or

meta:
  description: "floatplane data artist"
  url: "example.com"
  twitter: "yourname"
image: social.png
```

Using `meta.url/image` is a great default, but many users will want to use an alternative image sized for social media previews.